src/stylesheets/elements/form-controls/_checkbox-and-radio.scss

```diff
@@ -1,3 +1,131 @@
+@mixin format-input {
+  & + [class$="__label"]::before {
+    @content;
+  }
+}
+
+@mixin format-label {
+  & + [class$="__label"] {
+    @content;
+  }
+}
+
+@mixin checkbox-and-radio-colors(
+  $bg-color: $theme-input-background-color,
+  $checked-color: "default"
```

Comment:
...and the default `checked` color is either the dark or light link color set in settings.